Library/Homebrew/extend/os/linux/cask/installer.rb

```diff
@@ -13,7 +13,9 @@ module Installer
 
         sig { void }
         def check_stanza_os_requirements
-          raise ::Cask::CaskError, "macOS is required for this software."
+          raise ::Cask::CaskError, "macOS is required for this software." unless artifacts.reject do |k|
+            k.is_a?(::Cask::Artifact::Font)
+          end.empty?
```

Comment:
Is this equivalent logic? (The triple negative of `unless`/`reject`/`empty?` hurts my tiny 🧠)
```suggestion
          raise ::Cask::CaskError, "macOS is required for this software." if artifacts.any? do |k|
            k.is_a?(::Cask::Artifact::Font)
          end
```

Reply:
So, it needs to fail if it has anything other than a font. Because only fonts will be supported (in this PR)

Reply:
TY! In that case, this would be the idiomatic implementation:

```ruby
        def check_stanza_os_requirements
          return if artifacts.all?(::Cask::Artifact::Font)

          raise ::Cask::CaskError, "macOS is required for this software."
        end
```